.travis.yml

```diff
@@ -1,16 +1,35 @@
 language: ruby
 
-rvm:
-  - 2.4.3
-  - 2.5.0
+jobs:
+  include:
+    - stage: test-2.4
+      rvm: 2.4.3
+      script:
+        - bundle exec rake spec
 
-before_script:
-  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-  - chmod +x ./cc-test-reporter
-  - ./cc-test-reporter before-build
+    - stage: test-2.5
+      rvm: 2.5.0
+      before_script:
+        - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+        - chmod +x ./cc-test-reporter
+        - ./cc-test-reporter before-build
+      script:
+        - bundle exec rake spec
+      after_script:
+        - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
 
-script:
-  - bundle exec rake spec
+    - stage: tag-new-release-if-applicable
+      if: branch = master
+      before_script:
+        - openssl aes-256-cbc -K $encrypted_bfd358de8814_key -iv $encrypted_bfd358de8814_iv -in ./ci-helpers/atomic_cache_deploy.enc -out atomic_cache_deploy -d
+        - echo -e "Host example.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
+        - chmod 600 atomic_cache_deploy
+        - mv atomic_cache_deploy ~/.ssh/id_rsa
+        - git remote add deploy ssh://git@github.com/Ibotta/atomic_cache.git
```

Comment:
all this is because they don't support custom SSH keys outside of private repos for travis. this is the recommended way to do it for public repos. i used this [guide](https://scurker.com/automated-deploys-with-travis/).

Reply:
is the `encrypted_...` coming from travis env vars?

Reply:
Correct - that's the output from `travis encrypt`